.buildkite/react-native-pipeline.yml

```diff
@@ -55,6 +55,59 @@ steps:
             - exit_status: "*"
               limit: 1
 
+      - label: ':android: Build RN {{matrix}} native integration test fixture APK (Old Arch)'
+        key: "build-react-native-android-fixture-native-integration-old-arch"
+        timeout_in_minutes: 30
+        agents:
+          queue: macos-14
+        env:
+          JAVA_VERSION: "17"
+          NODE_VERSION: "18"
+          RN_VERSION: "{{matrix}}"
+          NOTIFIER_VERSION: '8.0.0'
+          BUILD_ANDROID: "true"
+          NATIVE_INTEGRATION: "1"
+        artifact_paths:
+          - "test/react-native/features/fixtures/generated/native-integration/old-arch/**/reactnative.apk"
+        plugins:
+          artifacts#v1.9.3:
+            upload: "test/react-native/features/fixtures/generated/native-integration/old-arch"
+            compressed: rn-projects.zip
```

Comment:
is this just here for debugging? these are pretty big bundles to upload on every build otherwise

Reply:
ah - this was copied inadevertently from another step - I think it was added in the error correlation PR but I guess it's not needed there either. I'll remove it from both 👍🏾